### ISSUE TITLE
[BETA] Problem with SNMP macro

### ISSUE THREAD

**Issue by sudoerss** [guillaumewatteeux/ansible-centreon#8]
Hello, I encounter a problem when I want to define the field Snmp Community via Macro, it is well taken into account however it is put in custom macro and not in the field 'SNMP Community & Version'

Would you have an idea?

thank you,

```
macros:
  - name: "$_HOSTSNMPCOMMUNITY$"
    value: "toto123587"
```

**Comment by guillaumewatteeux** [comment]
Need more parsing solution for macro.

**Comment by guillaumewatteeux** [comment]
fix in  94df81c
